src/client/Microsoft.Identity.Client/IConfidentialClientApplication.cs

```diff
@@ -76,6 +75,34 @@ public partial interface IConfidentialClientApplication : IClientApplicationBase
         /// </remarks>
         AcquireTokenOnBehalfOfParameterBuilder AcquireTokenOnBehalfOf(IEnumerable<string> scopes, UserAssertion userAssertion);
 
+        /// <summary>
+        /// Acquires an access token for this web API from the authority configured in the application,
+        /// in order to access another downstream protected web API on behalf of a user using the OAuth 2.0 On-Behalf-Of flow.
+        /// See https://aka.ms/msal-net-on-behalf-of.
+        /// This confidential client application was itself called with a token which will be provided in the
+        /// <paramref name="userToken">userToken</paramref> parameter.
+        /// </summary>
+        /// <param name="scopes">Scopes requested to access a protected API</param>
+        /// <param name="userToken">A JSON Web Token which was used to call the web API and contains the credential information
+        /// about the user on behalf of whom to get a token.</param>
+        /// <param name="longRunningProcessSessionKey">Key by which to look up the token in the cache.
+        /// If null, it will be set to the assertion hash by default.</param>
+        /// <returns>A builder enabling you to add optional parameters before executing the token request</returns>
+        AcquireTokenOnBehalfOfParameterBuilder InitiateLongRunningProcessInWebApi(IEnumerable<string> scopes, string userToken, ref string longRunningProcessSessionKey);
```

Comment:
Sadly you cannot modify an existing interface. It's a breaking change :( The solution is to add a new interface to the existing ones: 

```csharp
 public sealed partial class ConfidentialClientApplication
        : ClientApplicationBase,
            IConfidentialClientApplication,
            IConfidentialClientApplicationWithCertificate,
            IByRefreshToken
```

Reply:
`ILongRunningProcessWebApi` ? @jmprieur for naming ideas of the new interface